appveyor.yml

```diff
@@ -0,0 +1,24 @@
+version: "{branch} {build}"
+
+build:
+  verbosity: detailed
+
+# To avoid call-selfrun.bat FileNotFound exception, it installs and uses handle.exe as workaround provided by AppVeyor support.
+# see http://help.appveyor.com/discussions/problems/5975-the-process-cannot-access-the-file-because-it-is-being-used-by-another-process
+build_script:
+- gradlew.bat --info --no-daemon check
+
+cache:
+- C:\Users\appveyor\.gradle
+
+environment:
+  matrix:
+  - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
+  AZURE_ACCOUNT_NAME: sakamatest
+  AZURE_CONTAINER: blobcontainers
+  AZURE_CONTAINER_DIRECTORY: unittests_appveyor
+  AZURE_ACCOUNT_KEY:
+    secure: jrgzEWXOp3+c6r1clT8fRH9ck/aIdnBdVpqQWAwEH/t6w6mGzvBxIt2p0O7dMOowntuIEQBZZ3sIZXT1WzcgGLr11lShl73Y5c93NZJ9oZ7NK1vT4FXaWkRdT4jnAeIn
```

Comment:
Is it okay to include this credential (?) in the open-sourced code?

Reply:
Yes, this one is encrypted values set at [Encrypt configuration data page](https://ci.appveyor.com/tools/encrypt) at AppVeyor console and different from original one.
![encryption_data](https://user-images.githubusercontent.com/155269/47138172-a57b3080-d2f3-11e8-9403-275d151e8123.png)

Reply:
Got it. lgtm! (Maybe nice to have a comment to describe that, but I leave it to you.)

Reply:
Added comment and merged 6a5a62c